easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.13-foss-2021a-R-4.1.0.eb

```
@@ -0,0 +1,1058 @@
+        'checksums': ['f30d7eff4a1653fe96ace7f5e16f155e8cce011ccdd622f3c5e273b272d3d5f2'],
+    }),
+    ('seqLogo', '1.58.0', {
+        'checksums': ['9755621d3a8725356573c63b40a98e1bdb9ff650bdec089ccb6b190683d4a388'],
+    }),
+    ('rGADEM', '2.40.0', {
+        'checksums': ['f54b126363bd2b46ba9f56d3626a2efb9a25dac87b0048a69971168b255cb915'],
+    }),
+    ('MotifDb', '1.34.0', {
+        'checksums': ['7f2c8123ef0cbe7e12ce4670139cf3bdd5e96b14ccf95fd25b98055ecb7b9511'],
+    }),
+    ('motifStack', '1.36.0', {
+        'checksums': ['f78da2a47c8d8aa045d1be2da456fddd16e849677684547d34641f5e2722c37a'],
+    }),
+    ('ATACseqQC', '1.16.0', {
+        'checksums': ['b4cfc602a77aa6fc69567af995fdcda3770d9d6f2e5a1fd4952e7f09671f0f99'],
+    }),
+    ('ResidualMatrix', '1.2.0', {
+        'checksums': ['3becaaa3d668e4a64852b140118e6fc7b212d74c074d76ba554602ae5250fb5c'],
+    }),
+    ('batchelor', '1.8.0', {
+        'checksums': ['0a474dc558f73c26ad096e6f31a42d956fb916d334151d18ad75d4713901a2f6'],
+    }),
+    ('gsmoothr', '0.1.7', {
+        'checksums': ['b75ffd2a4a0f357762e02e46e355b45cc90ea637830f0a1b01f216bb4541e903'],
+    }),
+    ('Ringo', '1.56.0', {
+        'checksums': ['5de0e1baaf497b10fc5f496a881b3349abe6606b4d13dc6a28daf54e02dc51fb'],
+    }),
+    ('R.devices', '2.17.0', {
+        'checksums': ['c2d4b12949e0faab414c9f7575e39ee9dbf50ea12d858143df529f15ea365fb6'],
+    }),
+    ('R.filesets', '2.14.0', {
+        'checksums': ['707dd7c099aeadbecebbd2ba57e6400405aa44487c641f3729ce06c4c642bb23'],
+    }),
+    ('aroma.light', '3.22.0', {
+        'checksums': ['c50f8be7d7b6bafca440fdd54c957ac0c288904fd69115e996f13274033578fb'],
+    }),
+    ('PSCBS', '0.65.0', {
+        'checksums': ['3365065d5375c599eb024bfff12c5f6b10a6b1a4fe4ba6f200f7e83618dd399a'],
+    }),
+    ('aroma.core', '3.2.2', {
+        'checksums': ['35e0312e403a1894da036afac12d1d727c937e666d7eb400bfe4018b21244855'],
+    }),
+    ('R.huge', '0.9.0', {
+        'checksums': ['f12d113a965603cfdbbb46e1a57331d267916235555b22f2bbc06caa302ae58e'],
+    }),
+    ('aroma.apd', '0.6.0', {
+        'checksums': ['87513774baeed8e1d43f819e615c85278aef748ebc842f1ee50686c3392e37d1'],
+    }),
+    ('aroma.affymetrix', '3.2.0', {
+        'checksums': ['d279cf13cf6ed8369cfdf6682ace12924379bad578003453449aa1ccec31b0a7'],
+    }),
+    ('Rsolnp', '1.16', {
+        'checksums': ['3142776062beb8e2b45cdbc4fe6e5446b6d33505253d79f2890fe4178d9cf670'],
+    }),
+    ('Repitools', '1.38.0', {
+        'checksums': ['7464394ebd039cd1d57e242ddb71bf02fa35a6d0dc9aafae04c834e021c36a50'],
+    }),
+    ('BSgenome.Hsapiens.UCSC.hg38', '1.4.3', {
+        'checksums': ['c29911ca93935e3b9aa6a67e894337dc2ad707405da1500c9898bd0578519ecc'],
+    }),
+    ('MEDIPS', '1.44.0', {
+        'checksums': ['3128cdc286275faffc792c4a6cf08562cad072be6b9c8595d52353f06151a750'],
+    }),
+    ('RProtoBufLib', '2.4.0', {
+        'patches': ['%(name)s-%(version)s_fix-protobuf-build.patch'],
```

```suggestion
        'patches': ['RProtoBufLib-2.4.0_fix-protobuf-build.patch'],
```

The CI check expand `name` and `version` to the EC not the extenstion.